src/graphql/mutation/createMutationType.js

```diff
@@ -15,13 +15,13 @@ const createMutationType = schema =>
         schema
         .getProcedures()
         .filter(({ isMutation }) => isMutation)
-        .filter(procedure => !procedure.hasTableArg())
```

Comment:
The best solution for this might be to allow custom table types and hide the rest.